index.html

```diff
@@ -0,0 +1,20 @@
+    <p id="img-caption">This is an image of Dr Bashir</p>
+    </div>
+    <p id="tribute-info"> this is the paragraph which we need</p>
+    <a id="tribute-link" target="_blank"> This is a link</a>
```

Comment:
- KIndly Put this under a section tag as it is a sectionof your code

Reply:
- It is always good to use Grammarly when writing your comments so as to avoid some common spelling mistakes and grammatical errors such as you have done here 👍